README.md

```diff
@@ -18,40 +20,48 @@ The different parts of Cloudflow work in unison to dramatically accelerate your
+*Streamlet 3* then may store its data to some external system or just print to console.
+The example described here is a minimal Cloudflow application.
 The data sent between `Streamlets` is safely persisted in the underlying pub-sub system, allowing for independent lifecycle management of the different components.
 
-`Streamlets` can be scaled up and down to meet the load requirements of the application. 
+`Streamlets` can be scaled up and down to meet the load requirements of the application.
 The underlying data streams are partitioned to allow for parallelism in a distributed application execution.
 
-The `Streamlet` logic can be written using an extensible choice of streaming runtimes, such as Akka Streams and Spark. 
+The `Streamlet` logic can be written using an extensible choice of streaming runtimes, such as Akka Streams and Spark.
 The lightweight API exposes the raw power of the underlying runtime and its libraries while providing a higher-level abstraction for composing `streamlets` and expressing data schemas.
-Your code is written in your familiar Structured Streaming, Flink, or Akka Streams native API. 
+Your code is written in your familiar Structured Streaming, Flink, or Akka Streams native API.
 
 Applications are deployed as a whole. Cloudflow takes care of deploying the individual `streamlets` and making sure connections get translated into data flowing between them at runtime.
 
 ---
 
 ## Motivation: Why Cloudflow?
 
-Technologies like mobile, the Internet of Things (IoT), Big Data analytics, machine learning, and others are driving enterprises to modernize how they process large volumes of data. 
+Technologies like mobile, the Internet of Things (IoT), Big Data analytics, machine learning, and others are driving enterprises to modernize how they process large volumes of data.
 A rapidly growing percentage of that data is now arriving in the form of data streams.
-To extract value from that data as soon as it arrives, those streams require near-realtime processing. 
+To extract value from that data as soon as it arrives, those streams require near-realtime processing.
 We use the term "Fast Data" to describe applications and systems that deal with such requirements.
 
 The Fast Data landscape has been rapidly evolving, with tools like Spark, Flink, and Kafka Streams emerging from the world of large-scale data processing while projects like Reactive Streams and Akka Streams have emerged from the world of application development and high-performance networking.
 
-The demand for availability, scalability, and resilience is forcing Fast Data architectures to become more like microservice architectures. 
-Conversely, successful organizations building microservices find their data needs grow with their organization while their data sources are becoming more stream-like and more real-time. 
+The demand for availability, scalability, and resilience is forcing Fast Data architectures to become more like microservice architectures.
```

Comment:
```suggestion
The demand for availability, scalability, and resilience is forcing fast data architectures to become more like microservice architectures.
```

Reply:
This was already there but I can change it.